config-fatcrm-version.h.cmake

```diff
@@ -1,4 +1,9 @@
-#define FATCRM_VERSION_MAJOR ${FATCRM_VERSION_MAJOR}
-#define FATCRM_VERSION_MINOR ${FATCRM_VERSION_MINOR}
-#define FATCRM_VERSION_PATCH ${FATCRM_VERSION_PATCH}
-#define FATCRM_VERSION_STRING "${FATCRM_VERSION_STRING}"
+#ifndef CONFIG_FATCRM_H
```

Comment:
missing _VERSION in the ifndef/define, in case there's a config-fatcrm.h one day.